rerun_py/rerun_sdk/rerun/__init__.py

```diff
@@ -266,6 +267,25 @@ def init(
 
     For more advanced use cases, e.g. multiple recordings setups, see [`rerun.new_recording`][].
 
+    !!! Warning
+        If you don't specify a `recording_id`, it will default to a random value that is generated once
+        at the start of the process.
+        That value will be kept around for the whole lifetime of the process, and even inherited by all
+        its subprocesses, if any.
+
+        This makes it trivial to log data to the same recording in a multiprocess setup, but it also means
+        that the following code will _not_ create two distinct recordings:
+        ```
+        rr.init("my_app")
+        rr.init("my_app")
+        ```
+
+        To create distinct recordings from the same process, specify distinct recording IDs:
+        ```
+        rr.init("my_app", recording_id="recording1")
+        rr.init("my_app", recording_id="recording2")
```

Comment:
it would be nice to export a `rerun::random_recording_id()` and refer to it here